sphinx/ext/autosummary/__init__.py

```diff
@@ -502,6 +503,20 @@ def import_by_name(name, prefixes=[None]):
     raise ImportError('no module named %s' % ' or '.join(tried))
 
 
+def _import_or_importerror(modname):
```

Comment:
How about renaming this to `import_module`? I think the function name should represent its purpose, not behavior.